web/components/MetaTags/MetaTags.tsx

```diff
@@ -33,22 +35,25 @@ export const MetaTags = ({
       description={description}
       canonical={urlJoin('https://structuredcontent.live', canonicalPath)}
       noindex={noIndex}
-      openGraph={
-        image
-          ? {
-              images: [
-                {
-                  url: imageUrlFor(image)
-                    .ignoreImageParams()
-                    .size(1260, 630)
-                    .url(),
-                },
-              ],
-            }
-          : undefined
-      }
+      openGraph={{
+        images: [
+          image
+            ? {
+                url: imageUrlFor(image)
+                  .ignoreImageParams()
+                  .size(1260, 630)
+                  .url(),
+                alt: image.alt,
+              }
+            : {
+                url: '/static/images/opengraph-image.svg',
```

Comment:
Referring to item 2 in the previous comment: I believe this should be `opengraphImage.url` instead, in order to use the automatically generated "cache-breaking" URL (once the file is moved out of `static`).

Reply:
Of course! I couldn't immediately find out how to get such an URL, but of course next supports in in the form of `.src`. :100: Also fixed in 9ee794a.